### ISSUE TITLE
Control bug in companion app

### ISSUE THREAD

**Issue by dapper27** [Kemmey/Kemmey-TeslaWatch-Public#195]
I'm on build 331 but about to update. Have noticed this bug with control layout in companion app. Also, in the Watch app I had to have three attempts at clicking "Install" on the watch app before it installed - just didn't work before. Timed out with the install progress circle rotating for several minutes (I didn't watch it the whole time but just kept retrying and it eventually installed. Maybe just took time to uninstall all the remnants of the previous app)

![image](https://user-images.githubusercontent.com/44276659/97854727-c273d800-1cf1-11eb-9ed5-30b870955c94.png)

**Comment by Kemmey** [comment]
Yeah - the ios app is most certainly not ready for any sort of release - I just had to get a build out with the current login issues, and didn't have time to branch out, as the companion is not essential :-D

I'll keep working on the ios app at a slow pace - it might ship with 1.0.8, it might not - depends on how stable I can get it. But it will definitely ship as a login-assistant, as I get way too many support requests with login issues. Should also help with app re-install issues, as now there will be an ios app, and so the app can be re-installed from the ios app store app, and not only from the watch app store app...

**Comment by dapper27** [comment]
Thanks, thought as much! The settings on the watch app still work as before, although it's quite difficult in practice to move the different controls around into a different order as the screen scrolls very quickly when reaching the top or bottom of the watch!

**Comment by Kemmey** [comment]
Yeah - that picker certainly is not ideal. I had hoped watchOS 7 would bring a native time picker - no such luck. I'll have to polish it at some point - but it's a pretty obscure feature to begin with, so not the highest priority. 

On iOS there IS a native control though - so that's going to be much easier.

**Comment by dapper27** [comment]
Thanks for your continuing hard work and trying to overcome the obstacles that Apple no doubt keep putting in your way! Still the best Tesla Apple Watch app by a long way 🙂
